examples/notebooks/IAM_Handwriting_Recognition.ipynb

```diff
@@ -0,0 +1,5589 @@
+            "  inflating: data/testset.txt        \n",
+            "  inflating: data/trainset.txt       \n",
+            "  inflating: data/validationset1.txt  \n",
+            "  inflating: data/validationset2.txt  \n",
+            "\\nDownloading images ...\n",
+            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+            "100 1561M  100 1561M    0     0  16.1M      0  0:01:36  0:01:36 --:--:-- 15.7M\n",
+            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+            "100 1412M  100 1412M    0     0  15.9M      0  0:01:28  0:01:28 --:--:-- 7884k\n",
+            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+            "100 1436M  100 1436M    0     0  14.9M      0  0:01:36  0:01:36 --:--:-- 20.0M\n",
+            "\\nDone!\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bdCgkTtVlXb9"
+      },
+      "source": [
+        "IAM Handwriting Database consists of handwritten English text. Let's take a look at few examples from trainset."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 402
+        },
+        "id": "1njrXDrylwY5",
+        "outputId": "38a9c06f-99a5-40e8-8bac-068e2c125129"
+      },
+      "source": [
+        "num_examples = 3\n",
+        "\n",
+        "def resize_image(src_img, size=(64,64), bg_color=\"white\"): \n",
+        "    from PIL import Image\n",
+        "\n",
+        "    # rescale the image so the longest edge is the right size\n",
+        "    src_img.thumbnail(size, Image.ANTIALIAS)\n",
+        "\n",
+        "    # Create a new image of the right shape\n",
+        "    new_image = Image.new(\"RGB\", size, bg_color)\n",
+        "\n",
+        "    # Paste the rescaled image onto the new centered background\n",
+        "    new_image.paste(src_img, (int((size[0] - src_img.size[0]) / 2), int((size[1] - src_img.size[1]) / 2)))\n",
+        "\n",
+        "    # return the resized image\n",
+        "    return new_image\n",
+        "\n",
+        "with open(\"data/sentences.txt\") as f:\n",
+        "  lines = f.readlines()\n",
+        "while num_examples:\n",
+        "  sample = random.choice(lines)\n",
+        "  cols = sample.split()\n",
+        "  if cols[2] == \"err\":\n",
+        "    continue \n",
+        "  img_file = \"-\".join(cols[0].split(\"-\", 2)[:2])\n",
+        "  img = PIL.Image.open(f\"data/{name}.png\")\n",
```

Comment:
is `name` the right variable? You may want to re-run this notebook from scratch to make sure everything works..